.github/workflows/agent-reusable.yml

```
@@ -0,0 +1,26 @@
+        required: false
+        type: string
+        default: "1200"
+    secrets:
+      ANTHROPIC_API_KEY:
+        required: false
+      OPENAI_API_KEY:
+        required: false
+
+jobs:
+  call:
+    uses: ./.github/workflows/agent-orchestrate.yml
```

actionlint (1.7.7)

error while parsing reusable workflow "./.github/workflows/agent-orchestrate.yml": "workflow_call" event trigger is not found in "on:" at line:3, column:3

(workflow-call)